galasa-ui/src/components/test-runs/DateTimePicker.tsx

```diff
@@ -53,15 +71,32 @@ export default function DateTimePicker({
     }
   };
 
+  const fullLocale = preferences?.locale || 'en-US';
+
+  // Look up the required Flatpickr format
+  const datePickerFormat = LOCALE_TO_FLATPICKR_FORMAT_MAP[fullLocale] || 'm/d/Y';
+  
+  // The `locale` prop in DatePicker expects a short language code (e.g., 'en', 'fr') for calendar translation.
+  const languageCodeForPicker = fullLocale.split('-')[0];
+
+  // Generate a placeholder from the Flatpickr format string.
```

Comment:
What is a place holder ? The default current day/time ? `placeHolder` seems like it doesn't mean much.